ludwig/schema/metadata/configs/features.yaml

```diff
@@ -166,7 +166,30 @@ bag:
                 important rare tokens are, 90%, 95% or 99% of the number of tokens
                 will leave out only very rare tokens that should not influence performance
                 substantially
-            ui_display_name: Most common (vocabulary size)
+        most_common_percentile:
```

Comment:
I'm not a fan of duplicating code or documentation as it drastically increases the maintenance difficulty and the chance of drift.

Suggestion: Rewrite the metadata to be feature-type generic, decide on once place where the metadata should live (perhaps here, or in a new file i.e. `shared.yaml`), and remove all other copies.

Reply:
Made the change